shared/src/types.rs

```diff
@@ -570,7 +567,10 @@ impl<'a> PeerDiff<'a> {
         }
 
         // We won't update the endpoint if there's already a stable connection.
-        if !Self::peer_recently_connected(&old_info) {
+        if old_info
+            .map(|info| !info.is_recently_connected())
+            .unwrap_or(true)
```

Comment:
What is easier for your brain?
```suggestion
        if !old_info
            .map(|info| info.is_recently_connected())
            .unwrap_or_default()
```

Reply:
I like yours better.